assets/js/blocks/active-filters/block.tsx

```diff
@@ -413,15 +409,15 @@ const ActiveFiltersBlock = ( {
 	);
 };
 
-ActiveFiltersBlock.propTypes = {
+interface ActiveFiltersBlockProps {
 	/**
 	 * The attributes for this block.
 	 */
-	attributes: PropTypes.object.isRequired,
+	attributes: Attributes;
 	/**
 	 * Whether it's in the editor or frontend display.
 	 */
-	isEditor: PropTypes.bool,
-};
+	isEditor: IsEditor;
```

Comment:
Given that `assets/js/blocks/active-filters/types.ts` contains the type definitions, we could remove this part entirely and use the definitions from `assets/js/blocks/active-filters/types.ts` directly.

On https://github.com/woocommerce/woocommerce-blocks/pull/9613/files, you can find an example of this pattern.